### ISSUE TITLE
[ARM64][Compat][Win32][CubePDF Japanese & 3.0.0] : Unable to view the Print dialog box after clicking on "Print"

### ISSUE THREAD

**Issue by Michellehdr** [cube-soft/cube.pdf#38]
[ARM64]
[CubePDF Japanese & 3.0.0]
[Windows11]
Repro Steps:
1. Deploy latest Zinc/Germanium OS on ARM64 device. 
2. Perform Windows and Store updates. 
3. Install all the pre-requisites on your ARM64 device. X64-Graphics driver, ARM64X VC++ Redistributable, OpenCL™ and OpenGL® Compatibility Pack 
4. Install the application from setup file 
5. Launch the application
6. Click on three dots available in the browser after redirecting.
7. choose printer option.
8. Select cubepdf from printer dropdown.
9. Click on print.
10. Observe. Unable to view the Print dialog box after clicking "Print".

**Comment by clown** [comment]
As for the Arm64 version, several fixes have been applied in the latest version.
Please try the latest version to see if it solves the issue.
https://www.cube-soft.com/cubepdf/confirm.php?mode=arm64

If the issue persists, please attach the log file in the folder below.
C:\ProgramData\CubeSoft\CubePdf\Log

**Comment by Michellehdr** [comment]
@clown our tester verified on the newer build, and we found the issue still repro. Could you please take a look at the bug again?
Validated the application with the latest version - 3.4.1 in SPX.

**Comment by clown** [comment]
Could you attach the log files in the following folder?
C:\ProgramData\CubeSoft\CubePdf\Log

**Comment by Michellehdr** [comment]
@clown unfortunately, we are not allowed to paste log outside of Microsoft bug management tool. Please let me know how I can help except the log. Thanks a lot!

**Comment by clown** [comment]
OK. I will do some more research. Thanks.

**Comment by Michellehdr** [comment]
@clown just would like to follow up with you: is there any progress on the investigation? Any other information needed?

**Comment by clown** [comment]
At this time, similar problems have not been reported, including in the development environment, so we are unable to determine the cause of the problem.

We expect that there is some kind of problem with CubeProxy.exe in the situation you described. Are there any errors in the following log file that you are concerned about? (as far as you can describe on this issue is OK)
C:\ProgramData\CubeSoft\CubePdf\Log\CubeProxy.log

**Comment by Michellehdr** [comment]
@clown did you try with ARM64 for this bug? This is an ARM64 specific issue.

**Comment by clown** [comment]
Yes, I have tried it on an ARM64 device (Surface Pro X) and confirmed that it works fine in that environment. So it seems that it is not happening on all ARM64 devices.

**Comment by Michellehdr** [comment]
@clown our tester tried again on different Windows versions, we could not find a workable App version. We tried on the following device. Do you think it could be HW related?
X13s Lenovo with a 8cx gen3. (latest im-market generation)

**Comment by clown** [comment]
We have released CubePDF 4.0.0, which is based on NET Framework 4.8 instead of NET 6.0.
Please try again with the latest version.
https://www.cube-soft.com/cubepdf/
https://en.cube-soft.jp/entry/2024/07/10/cubepdf-4.0.0-or-later
